Doc link to MDE system requirements.  Prevent drift and confusion. (#3480)
MicrosoftDocs/memdocs: memdocs/intune/protect/advanced-threat-protection.md

You can integrate Microsoft Defender for Endpoint with Microsoft Intune as a Mobile Threat Defense solution. Integration can help you prevent security breaches and limit the impact of breaches within an organization.

> Go here to check system requirements for Microsoft Defender for Endpoint:  [Click Here](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/minimum-requirements?view=o365-worldwide "Microsoft Defender for Endpoint System Requirements"

Reviewer: ```suggestion
For the system requirements for Microsoft Defender for Endpoint, see [Minimum requirements for Microsoft Defender for Endpoint](/microsoft-365/security/defender-endpoint/minimum-requirements).
```

> 

Reviewer: ```suggestion
```
